src/mongoModelsRepo/createPersona.ts

```diff
@@ -8,20 +8,20 @@ import { PERSONAS_COLLECTION } from './utils/constants/collections';
 export default (config: Config) => {
   return async (opts: CreatePersonaOptions): Promise<CreatePersonaResult> => {
     const collection = (await config.db).collection(PERSONAS_COLLECTION);
-
-    const docToInsert = {
-      name: opts.name,
-      organisation: new ObjectID(opts.organisation),
-    };
+    const personaId = new ObjectID();
```

Comment:
Generating this ahead of time and using it as the name is a smart solution, nicely done 👍

Reply:
@ryansmith94 I'm not sure it's a good idea to drop the id into the name field on the db. I feel a name field should hold the name, and the change should be just done in the ui?

Reply:
Yeah I totally agree, If I remember correctly we had discussed that as the preferred option but I think it was difficult to implement as it would require changes to the pipelines when personas were chosen.

Reply:
@cbishopvelti